content/en/docs/reference/glossary/api-group.md

```diff
@@ -0,0 +1,18 @@
+---
+title: API Group
+id: api-group
+date: 2019-09-02
+full_link: /docs/concepts/overview/kubernetes-api/#api-groups
+short_description: >
+  API group makes it easier to extend the Kubernetes API. The API group is specified in a REST path and in the `apiVersion` field of a serialized object.
+
+aka:
+tags:
+- fundamental
+- architecture
+---
+API group makes it easier to extend the Kubernetes API. The API group is specified in a REST path and in the `apiVersion` field of a serialized object.
```

Comment:
Here's the test that I have in mind for the glossary entry:
Is it possible to take that glossary definition and insert it (via a Hugo shortcode) into https://kubernetes.io/docs/concepts/overview/kubernetes-api/#api-groups
so that it makes sense to the reader?

How about adding text along the lines of “In the Kubernetes API, an _API group_ is a set of related API paths. You can enable or disable each API group by changing the configuration of your API server. You can also disable or enable paths to specific resources.”?

Reply:
```suggestion
In the Kubernetes API, an API group is a set of related API paths. You can enable or disable each API group by changing the configuration of your API server. You can also disable or enable paths to specific resources.
```

If we add glossary definition to https://kubernetes.io/docs/concepts/overview/kubernetes-api/#api-groups, it will be self pointing link since full_link is https://kubernetes.io/docs/concepts/overview/kubernetes-api/#api-groups.